public/js/react-elements.js

```diff
@@ -94,57 +94,56 @@ function CompletedLocationList () {
         {GetCompleteLocations()}
       </tbody>
     </table>
-  )
+  }
 }
-
-function Dungeon (props) {
-  return (
-    <tr onClick={() => ChangeStatus(props)}>
-      <td>{props.name}</td>
-      <td>{props.status}</td>
+class Dungeon extends React.Component {
+  render() {
+    <tr onClick={() => ChangeStatus(this.props)}>
+      <td>{this.props.name}</td>
+      <td>{this.props.status}</td>
     </tr>
-  )
+  }
 }
 
-function World (props) {
-  return (
-    <tr onClick={() => ReactDOM.render(<WorldItemList id={props.id} />, document.getElementById('avaliable-root'))}>
-      <td>{props.name}</td>
-      <td>{props.items}</td>
+class World extends React.Component {
+  render() {
+    <tr onClick={() => ReactDOM.render(<WorldItemList id={this.props.id} />, document.getElementById('avaliable-root'))}>
+      <td>{this.props.name}</td>
+      <td>{this.props.items}</td>
     </tr>
-  )
+  }
 }
 
-function Item (props) {
-  return (
+class Item extends React.Component {
+  return() {
     <tr>
-      <td>{props.location}</td>
-      <td>{props.name}</td>
+      <td>{this.props.location}</td>
+      <td>{this.props.name}</td>
     </tr>
-  )
```

Comment:
Should be render() rather than return()

Reply:
My bad, I missed this. I've just pushed the updates, kindly review.